src/test/rpc_tests.cpp

```diff
@@ -75,6 +75,7 @@ BOOST_AUTO_TEST_CASE(rpc_rawparams)
     BOOST_CHECK_THROW(CallRPC("sendrawtransaction null"), std::runtime_error);
     BOOST_CHECK_THROW(CallRPC("sendrawtransaction DEADBEEF"), std::runtime_error);
     BOOST_CHECK_THROW(CallRPC(std::string("sendrawtransaction ")+rawtx+" extra"), std::runtime_error);
+    BOOST_CHECK_THROW(CallRPC(std::string("dumptxoutset utxo.dat`")), std::runtime_error);
```

Comment:
Why does this generate a runtime error?
I might be wrong but I think this test is for utility (no side-effects) RPCs only; no utxo set has been set up here, so I think this is unsafe

Reply:
Well, I am unsure *why* it generates a runtime error, but I actually expected it to be generated because the file name is wrong. However, if a runtime error should not happen here, maybe I should look into it.

Reply:
Isn't ` a valid filename character on pretty much all OSes?

That was not really my point though; I don't think this is the place to test this. The functional test is. I'd just remove this line.

Reply:
Not sure if all OSes accept a backtick, but because I'm using boost's **portable_name** function it behaves more strictly towards non-portable file names. Here's the whole list of options that I've been using: https://www.boost.org/doc/libs/1_57_0/libs/filesystem/doc/portability_guide.htm
We could go less or more strict, of course. 
I will then remove this test.

Reply:
I think that's very much overkill. The filename only needs to be valid for the OS bitcoind is running on. Looking at that table, `portable_posix_name` (which is implied by `portable_name`) rules out using non-ASCII characters which is not good for internationalization!

Reply:
Well, then I could use function *native* for that. My idea was to have a check that takes care of allowing only such names where one could create a file that could be copied to/from any OS. But, if this is too much I will make it less strict.

Reply:
The problem is that boost's definition of "any OS" is way too broad. It would at most need to be compatible with modern operating systems that bitcoind runs on, not 80's UNIX or DOS.
In any case, `.native` is better here.